modules/ROOT/pages/installation-prerequisites.adoc

```diff
@@ -6,34 +6,71 @@ endif::[]
 
 To install Anypoint API Community Manager you will need:
 
-. An Anypoint admin user account.
-. A Salesforce API Community Manager organization admin user with a role.
-+
+== A Salesforce Admin User
+
+Installation requires a Salesforce API Community Manager organization admin user with a role.
+
 To update your user role and permissions:
 
-.. Log in to your Salesforce organization.
-.. In *Setup*, search for *Users* using the *Quick Find* box and click *Users*.
-.. Locate your username and click *Edit*.
-.. Set the *Role* for your account.
+. Log in to your Salesforce organization.
+. In *Setup*, search for *Users* using the *Quick Find* box and click *Users*.
+. Locate your username and click *Edit*.
+. Set the *Role* for your account.
```

Comment:
How does the user "set" the role? Perhaps specify something like "Set the role for your account by selecting from the *Role* list"?